server.go

```diff
@@ -632,8 +633,31 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB, cc *chainControl,
 		return link.Bandwidth()
 	}
 
+	// Instantiate mission control with config from the sub server.
+	//
+	// TODO(joostjager): When we are further in the process of moving to sub
+	// servers, the mission control instance itself can be moved there too.
+	mcCfg := &routing.MissionControlConfig{
+		MinProbability:        routing.DefaultMinProbability,
+		PaymentAttemptPenalty: routing.DefaultPaymentAttemptPenalty,
+		PenaltyHalfLife:       routing.DefaultPenaltyHalfLife,
+	}
+
+	mcRefl := reflect.ValueOf(cfg.SubRPCServers.RouterRPC).Elem()
+	if mcRefl.NumField() != 0 {
```

Comment:
possible to define a method `GetMCConfig` in `routerRPC` instead, to avoid the reflection?

Reply:
Ok, good suggestion. Although in general I still haven't heard a convincing argument for the conditional compilation. I only know that it cost me a lot time in total dealing with it, all spread out over hundreds of micro-penalties. I am sure users are experiencing the same.

Reply:
I'm not fundamentally opposed to conditional compilation, but reflection degrades our beautiful language to javascript.

Reply:
I am neither, but in the way we use it, I don't see what we gain. I hope in the future there will be a majority to remove it in favor of clearly marking things as experimental in the rpc docs.